js/client/modules/@arangodb/testutils/testing.js

```diff
@@ -501,6 +501,9 @@ function iterateTests(cases, options) {
   let results = {};
   let cleanup = true;
 
+  if (options.extremeVerbosity === true) {
```

Comment:
Why do we compare `options.extremeVerbosity` to the boolean `true` value here, whereas in other cases in this PR we only check if the value is truey or falsy?
In the other places in this PR we only do `if (options.extremeVerbosity)` and `if (!options.extremeVerbosity)`.

Reply:
fixed.